internal/gatewayapi/validate.go

```diff
@@ -534,6 +534,108 @@ func (t *Translator) validateTerminateModeAndGetTLSSecrets(listener *ListenerCon
 	return secrets
 }
 
+func (t *Translator) validateAndGetFrontendValidationCACerts(listener *ListenerContext, resources *Resources) []byte {
+	var cacerts []byte
+	for _, ref := range listener.TLS.FrontendValidation.CACertificateRefs {
```

Comment:
is it possible to dedup this logic and share a common method with https://github.com/envoyproxy/gateway/blob/828edfb1cf320fd56002e29ddc01aa639cf71265/internal/gatewayapi/clienttrafficpolicy.go#L791 ?